corehq/apps/export/models/new.py

```diff
@@ -1624,15 +1624,17 @@ def _add_export_items_for_cases(cls, app, xform_schema, forms, xform):
                 actions = form.actions.get_open_subcase_actions()
             else:
                 actions = form.actions.subcases
-            for i, action in enumerate(actions):
-                action.subcase_index = i
+                for i, action in enumerate(form.actions.subcases):
+                    action.form_element_name = 'subcase_{}'.format(i)
+
+            for action in actions:
                 if action.repeat_context:
                     repeats_with_subcases.append(action)
                 else:
                     non_repeating_subcases.append(action)
 
         for subcase_action in non_repeating_subcases:
-            root_path = "/data/subcase_{}".format(subcase_action.subcase_index)
+            root_path = "/data/{}".format(subcase_action.form_element_name)
```

Comment:
yea, not crazy about it either (though that's how it get's done when building the XML). It's hard to add it as a method on the action since the action would need to know it's index in the list. We do have a way of doing that - I'll take a look.

Reply:
e6b727f